examples/pos-mainchain/config/default/genesis_assets_with_preset.json

```diff
@@ -0,0 +1,3074 @@
+{
+	"assets": [
+		{
+			"module": "interoperability",
+			"data": {
+				"outboxRootSubstore": [
+					{
+						"storeKey": "04000000",
+						"storeValue": {
+							"root": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"
+						}
+					},
+					{
+						"storeKey": "04000001",
+						"storeValue": {
+							"root": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"
+						}
+					}
+				],
+				"chainDataSubstore": [
+					{
+						"storeKey": "04000000",
+						"storeValue": {
+							"lastCertificate": {
+								"height": 0,
+								"timestamp": 0,
+								"stateRoot": "3401831cdf0952ee7a513008bf027f6064967d5eadf964d51b9deb7d255e10d1",
+								"validatorsHash": "43538f898219b129efd9d96eba7de850f5544ee9ee3a7d9b942e8df09ed24903"
+							},
+							"name": "lisk_mainchain",
+							"status": 1
+						}
+					},
+					{
+						"storeKey": "04000001",
+						"storeValue": {
+							"lastCertificate": {
+								"height": 0,
+								"timestamp": 0,
+								"stateRoot": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
+								"validatorsHash": "43538f898219b129efd9d96eba7de850f5544ee9ee3a7d9b942e8df09ed24903"
+							},
+							"name": "sidechain1",
+							"status": 0
+						}
+					},
+					{
+						"storeKey": "04000002",
+						"storeValue": {
+							"lastCertificate": {
+								"height": 0,
+								"timestamp": 0,
+								"stateRoot": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
+								"validatorsHash": "43538f898219b129efd9d96eba7de850f5544ee9ee3a7d9b942e8df09ed24903"
+							},
+							"name": "sidechain2",
+							"status": 2
+						}
+					}
+				],
+				"channelDataSubstore": [
+					{
+						"storeKey": "04000000",
```

Comment:
I tried to exclude mainchain from the genesis asset at the first place, however this will show up when creating the blob
```
Error: If a chain account for another sidechain is present, then a chain account for the mainchain must be present, as well as the own chain account.

```

Reply:
```
// lisk-sdk/framework/src/modules/interoperability/base_interoperability_module.ts
		if (
			hasSidechainAccount &&
			!(chainDataStoreKeySet.has(mainchainID) && chainDataStoreKeySet.has(ctx.chainID))
		) {
			throw new Error(
				'If a chain account for another sidechain is present, then a chain account for the mainchain must be present, as well as the own chain account.',
			);
		}
```

Reply:
this is true only on a sidechain